src/main/java/frc/robot/Controls.java

```diff
@@ -135,6 +138,13 @@ private void configureDrivebaseBindings() {
                 .runOnce(() -> s.drivebaseSubsystem.seedFieldCentric())
                 .withName("Reset gyro"));
     s.drivebaseSubsystem.registerTelemetry(logger::telemeterize);
+    var swerveCoastButton =
+        Shuffleboard.getTab("Controls")
+            .add("Swerve Coast Mode", false)
+            .withWidget(BuiltInWidgets.kToggleButton)
+            .getEntry();
+    new Trigger(() -> swerveCoastButton.getBoolean(false))
+        .whileTrue(s.drivebaseSubsystem.coastMotors());
```

Comment:
What sets the motors back to brake mode (either while disabled, and especially when the robot is enabled for teleop or auto)?